js/index.js

```diff
@@ -40,3 +40,215 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+// Add Call to Action Image
+let ctaImg = document.getElementById("cta-img");
+ctaImg.setAttribute('src', siteContent.cta['img-src']);
+
+// Simplified selector function
```

Comment:
This is amazing. You are really taking the power of javaScript to make your life easier.